lib/ramble/docs/tutorials/hello_world.rst

```diff
@@ -0,0 +1,219 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _hello_world_tutorial:
+
+=====================================================
+1) Getting Started Running A "Hello World" Experiment
+=====================================================
+
+This tutorial will provide a basic introduction to navigating Ramble and running
+experiments. In this tutorial, you will set up and run a basic experiment using
+``hostname``.
```

Comment:
I think because hostname can mean lots of different things to different people, we might want to call out that it's a basic linux utilty that we're using as our application?